### PR TITLE
Install udf

### PR DESCRIPTION
**Pull request by nzarayeneh**

### CODE REVIEW

**Review by parkererickson — CHANGES_REQUESTED** [review]
Looks like there was a merge conflict. These need to be resolved before merge.

File: pyTigerGraph/gds/featurizer.py [hunk 1]
```diff
@@ -223,7 +231,11 @@ def _install_query_file(self, query_name: str, replace: dict = None, query_path:
         if query_name == "tg_fastRP":
             # Drop all jobs on the graph
             self.conn.gsql("USE GRAPH {}\n".format(self.conn.graphname) + "drop job *")
+<<<<<<< HEAD
```
Comment (tg-bill) [inline pyTigerGraph/gds/featurizer.py:234]: And here as well

**Comment by nzarayeneh** [comment]
> Looks like there was a merge conflict. These need to be resolved before merge.

yeah, I am going to fix it

**Comment by parkererickson** [comment]
Can you specify that `global_schema` in `installAlgorithm` is only needed for FastRP? just want to have clarity in where to use it.

**Review by tg-bill — APPROVED** [review]

**Pull request merged**